### ISSUE TITLE
Mega Split not working ...

### ISSUE THREAD

**Issue by Dramex** [Barbosik/MultiOgar#324]
related to #162

there a delay in playerRecombineTime = 0, like 1 second, its not real instant merge
we can't do mega split like : 
https://www.youtube.com/watch?v=6mxTeuSegA8
https://www.youtube.com/watch?v=6mmhgg0FOT0

the problem was on OGARUL they solved it :
https://github.com/AJS-development/Ogar-unlimited/pull/1208

waiting for fix on MultiOgar

**Comment by Barbosik** [comment]
yes, this delay was added because without it big cell will not be able to split at all - it will be remerged just after split

**Comment by Dramex** [comment]
Multi split will fix this, could you see the video, and try to add option for that.
The mega split make the people show there professionality,
its can be as gamemode if its cause anything on FFA.

**Comment by Barbosik** [comment]
you can remove it by remove the following two code blocks:
1) https://github.com/Barbosik/MultiOgar/blob/master/src/GameServer.js#L873-L877
2) https://github.com/Barbosik/MultiOgar/blob/master/src/GameServer.js#L903-L907

**Comment by Dramex** [comment]
still there some collision 
164.132.200.92:2016
http://agar.io/?ip=164.132.200.92:2016

**Comment by Barbosik** [comment]
@Dramex: remove also this block: https://github.com/Barbosik/MultiOgar/blob/master/src/entity/PlayerCell.js#L16-L21

**Comment by Dramex** [comment]
near of it, but still there different than the viedo..
the cells move out the road of it, its must follow the mouse : (
